### PR TITLE
New docker files not working

### PR DESCRIPTION
**Pull request by ArimantasJakutonis**
Docker files that don't seem to work

### CODE REVIEW

**Comment by Karcikovas** [comment]
Create Pull Request to Develop not Master

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3'
+services: 
+  app:
+    container_name: docker-node
+    restart: always
+    build: .
+    ports:
+      - '80:3000'
```
Comment (Karcikovas) [inline docker-compose.yml]: No docker Images used?  😄

**Comment by Karcikovas** [comment]
Status?

**Comment by ArimantasJakutonis** [comment]
@Karcikovas been lazy with this :disappointed: . Will try to do study about Docker some more and figure out how to make it work today if possible.

**Pull request closed without merging**